winlogbeat/checkpoint/file_windows.go

```diff
@@ -29,8 +30,8 @@ func createWriteThroughFile(path string) (*os.File, error) {
 		syscall.GENERIC_READ|syscall.GENERIC_WRITE,               // Access Mode
 		uint32(syscall.FILE_SHARE_READ|syscall.FILE_SHARE_WRITE), // Share Mode
 		nil, // Security Attributes
-		syscall.CREATE_ALWAYS,                                          // Create Mode
-		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH), // Flags and Attributes
+		syscall.CREATE_ALWAYS, // Create Mode
+		uint32(syscall.FILE_ATTRIBUTE_NORMAL|_FILE_FLAG_WRITE_THROUGH|_FILE_FLAG_NO_BUFFERING), // Flags and Attributes
```

Comment:
From what I understand on Windows you need to use `FILE_FLAG_WRITE_THROUGH` to get writes to bypass the cache. FILE_FLAG_NO_BUFFERING implies FILE_FLAG_WRITE_THROUGH too, but it affects reads and adds the requirement that your reads must be aligned to disk sector boundaries. So I don't think we need FILE_FLAG_NO_BUFFERING here.